pyEpiabm/pyEpiabm/routine/simulation.py

```diff
@@ -116,6 +135,50 @@ def configure(self,
             folder, filename,
             output_titles)
 
+        self.status_output = False
+        self.infectiousness_output = False
+        self.ih_status_writer = None
+        self.ih_infectiousness_writer = None
+
+        if inf_history_params:
+            # Setting up writer for infection history for each person. If the
+            # inf_history_params dict is empty then we do not need to record
+            # this
+            self.status_output = inf_history_params.get("status_output")
+
+            self.infectiousness_output = inf_history_params\
+                .get("infectiousness_output")
+            person_ids = []
+            person_ids += [person.id for cell in population.cells for person
+                           in cell.persons]
+            self.ih_output_titles = ["time"] + person_ids
+            ih_folder = os.path.join(os.getcwd(),
+                                     inf_history_params["output_dir"])
+
+            if self.status_output:
+
+                ih_file_name = "ih_status_output.csv"
```

Comment:
Veyr pernickety, but this is clearly an output file, so don't need to be labelled as such. `inf_status_history` and `infectiousness_history` might be more explanatory? But I'm not set on these, so if you prefer the current names then I am happy for you to keep them and resolve this comment

Reply:
Ah I see, yes I will change the names of these files